From d85ac8340f168da3a08c34c1dc0e399278f55d0e Mon Sep 17 00:00:00 2001
From: Jiri Moskovcak <jmoskovc@redhat.com>
Date: Wed, 10 Apr 2013 14:12:51 +0200
Subject: [LIBREPORT PATCH 08/11] added report-cli event for anaconda should
 help with rhbz#950544

- it's just a workaround so I'm not going to close the bz ticket by this commit

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/plugins/bugzilla_anaconda_event.conf | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/plugins/bugzilla_anaconda_event.conf b/src/plugins/bugzilla_anaconda_event.conf
index 6d888e4..a9df9df 100644
--- a/src/plugins/bugzilla_anaconda_event.conf
+++ b/src/plugins/bugzilla_anaconda_event.conf
@@ -1,3 +1,6 @@
 EVENT=report_Bugzilla component=anaconda
 	reporter-bugzilla -b \
 		-F /etc/libreport/plugins/bugzilla_format_libreport.conf
+
+EVENT=report-cli component=anaconda
+    report-cli -e report_Bugzilla -- "$DUMP_DIR"
-- 
1.8.1.4

